CHANGES.md

```diff
@@ -30,6 +30,8 @@ _Version 0.6.0_:
              (@ejgallego, report by @palmskog)
  * [sercomp] Add `check` mode to compilers to check all proofs without
              outputting `.vo`. (@palmskog)
+ * [sercomp] Add "hacky" `--quick` mode to skip opaque proofs.
```

Comment:
Isn't this an "option" rather than a "mode"? Or do we want to keep that distinction?

Reply:
Indeed that's more of an option, as it does apply to several modes; thanks, will amend. I guess I was influenced subconsciously by the internal Coq option being named `async_proofs_mode` :) 

IMHO we would want a bit of a different API for proof checking in general, so parsing would never check proofs by default, but for now we are forced to expose the internal Coq modes and a bit of a combinatorial mess.